Library/Formula/clang-format-vim.rb

```diff
@@ -0,0 +1,14 @@
+class ClangFormatVim < Formula
+  desc "Minimal clang-format vim-integration"
+  homepage "http://clang.llvm.org/docs/ClangFormat.html#vim-integration"
+  url "https://llvm.org/svn/llvm-project/cfe/trunk/tools/clang-format/clang-format.py"
```

Comment:
Well *$#;!. That commit should not be on this branch. Can I simply remove it and force-push, or does that break merging for the pull request?